opentech/apply/projects/views.py

```diff
@@ -82,6 +90,41 @@ def get_contracts(self):
         return contracts
 
 
+class PaymentsMixin:
+    def get_context_data(self, **kwargs):
+        project = self.get_object()
+
+        payments = {
+            'availabe_statuses': REQUEST_STATUS_CHOICES,
+            'not_rejected': project.payment_requests.exclude(status=DECLINED),
+            'rejected': project.payment_requests.filter(status=DECLINED),
+            'totals': self.get_totals(project),
+        }
+
+        context = super().get_context_data(**kwargs)
+        context['payments'] = payments
+        return context
+
+    def get_totals(self, project):
+        def percentage(total, value):
+            return round((value / total) * 100)
```

Comment:
We should be working in Decimal at this point - anything to do with money. 
`(value / total * 100).quantize(Decimal('1'), rounding=ROUND_DOWN)`

Reply:
Fixed and rebased.